manager/integration/tests/test_recurring_job.py

```diff
@@ -476,13 +479,96 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     13. Verify that during the recurring backup, the volume's frontend is
         disabled, and pod cannot start.
     14. Wait for the recurring backup finishes.
-        Delete the recurring backup
-    15. In a 6-mintute retry loop, verify that the pod can eventually start.
-
-    16. Change the setting allow-recurring-job-while-volume-detached to false
-    17. cleanup
+        Delete the recurring backup.
+    15. In a 6-minutes retry loop, verify that the pod can eventually start.
+    16. Change the setting allow-recurring-job-while-volume-detached to false.
+    17. Cleanup.
     """
-    pass
+    recurring_job_setting = \
+        client.by_id_setting(SETTING_RECURRING_JOB_WHILE_VOLUME_DETACHED)
+    client.update(recurring_job_setting, value="true")
+
+    vol = common.create_and_check_volume(client, volume_name, size=str(1 * Gi))
+
+    lht_hostId = get_self_host_id()
+    vol.attach(hostId=lht_hostId)
+    vol = common.wait_for_volume_healthy(client, vol.name)
+
+    data = {
+        'pos': int(SIZE),
```

Comment:
I am not sure why we choose `pos=SIZE`. I see this on some other places as well. Should `pos` be 0 because it is the location in the block device that we start writing to?

Reply:
Addressed.